<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="radius_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>radius:</string>
   </property>
  </widget>
  <widget class="QLabel" name="size_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="size_val_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="radius_input">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>25</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>calculate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
